src/common/services/request/query/fetch-plays.js

```diff
@@ -46,12 +46,22 @@ export function FetchPlaysBySlugAndUser(playslug, username) {
   return payload;
 }
 
-export function FetchPlaysByFilter(where_clause) {
+export function FetchPlaysByFilter(where_clause, sortBy) {
   const payload = { ...BasiFetchParam };
   if (where_clause) {
     payload.where = where_clause;
   }
 
+  if (sortBy === 'Newest') {
+    payload.orderBy = { created_at: 'desc' };
+  } else if (sortBy === 'Oldest') {
+    payload.orderBy = { created_at: 'asc' };
+  } else if (sortBy === 'Most Liked') {
+    payload.orderBy = { play_like_aggregate: { count: 'desc' } };
+  } else {
+    payload.orderBy = { description: 'desc' };
+  }
+
```

Comment:
we could use `ternary operator` instead of `if-else`
- ternary operator can make the code more concise
```
payload.orderBy = 
  sortBy === 'Newest' ? { created_at: 'desc' } :
  sortBy === 'Oldest' ? { created_at: 'asc' } :
  sortBy === 'Most Liked' ? { play_like_aggregate: { count: 'desc' } } :
  { description: 'desc' };

```

Reply:
Hey, it's not about making the code more concise, 
it's about making the code more readable and easily scalable.
Nested ternary is generally avoided.
You can go through this [article](https://medium.com/@benlmsc/stop-using-nested-ternary-operators-heres-why-53e7e078e65a) and [eslint rule](https://eslint.org/docs/latest/rules/no-nested-ternary)

Reply:
ok 👍🏻